src/inet.h

```diff
@@ -45,7 +51,6 @@ int inet_aton(const char *cp, struct in_addr *inp);
 #endif
 
 #ifdef LUASOCKET_INET_PTON
-const char *inet_ntop(int af, const void *src, char *dst, socklen_t cnt);
```

Comment:
`inet_ntop()` was removed by b1d1e721d1b325cacfc6342c696191730dea357d .
And `inet_ntop()` causes build error.

(We can resolve the build error just removing this line. `LUASOCKET_INET_PTON` with recent MinGW reports a warning but it's not an error.)